### PR TITLE
keep note is included in slack message when kept synchronously 

### PR DESCRIPTION
**Pull request by camhashemi**
(i.e. not the extension)

@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibrarySubscriptionCommander.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ class LibrarySubscriptionCommander @Inject() (
         case info: SlackInfo =>
           val keepTitle = if (keep.title.exists(_.nonEmpty)) { keep.title.get } else { "a keep" }
           val text = s"<http://www.kifi.com/${keeper.username.value}?kma=1|${keeper.fullName}> just added <${keep.url}|${keepTitle}> to the <http://www.kifi.com/${owner.username.value}/${library.slug.value}?kma=1|${library.name}> library."
-          val attachments: Seq[SlackAttachment] = keep.note.map { note => Seq(SlackAttachment(fallback = "", text = "\"" + note + "\" - " + keeper.firstName)) }.getOrElse(Seq.empty)
+          val attachments: Seq[SlackAttachment] = if (keep.note.exists(_.nonEmpty)) { Seq(SlackAttachment(fallback = "Check out this keep", text = "\"" + keep.note.get + "\" - " + keeper.firstName)) } else { Seq.empty }
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibrarySubscriptionCommander.scala]: `keep.note.toSeq.collect { case content if content.nonEmpty =>  SlackAttachment(fallback = "Check out this keep", text = s""""$content" - ${keeper.firstName}""") }`

**Comment by andrewconner** [comment]
Cool. So this is basically mobile right now?

**Comment by camhashemi** [comment]
yup. thank you Leo.

**Comment by camhashemi** [comment]
@andrewconner I also added the evolution change in there, cool to merge? evolutions passed in dev mode.

**Comment by andrewconner** [comment]
Yep

**Pull request merged**